_summaries/radford2021learning_1.md

```diff
@@ -1,19 +1,55 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: radford2021learning
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jchavezberkeley
+score: 8 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+The core idea begins with a model that is trained on image-text pairs, a computer vision task that is often restricts models to predict just a finite set of categories. This paper introduces CLIP (Contranstive Language-Image Pre-Training) which is a model that pre-trains, in natural langauge, on the text so that it predicts to "most relevant text snippet" for each image. 
+
+Turns out that pre-training on the text actually helps learn image representations better and more efficiently than before, allowing zero-shot transfer on other datasets without worry about redefining categories.
+
+
 * How is it realized (technically)?
```

Comment:
I would add a couple of sentences about the dataset they used, since it was created in order to train this model.